src/outline.jl

```diff
@@ -245,13 +245,11 @@ function str_value(x)
         return string("\"\"\"", x.val, "\"\"\"")
     elseif x.kind === CSTParser.Tokens.STRING
         return string("\"", x.val, "\"")
+    elseif x.typ === CSTParser.Parameters
+        return ";" * join(str_value(a) for a in x)
     elseif x.typ === CSTParser.IDENTIFIER || x.typ === CSTParser.LITERAL || x.typ === CSTParser.OPERATOR || x.typ === CSTParser.KEYWORD
         return CSTParser.str_value(x)
     else
-        s = ""
-        for a in x
-            s *= str_value(a)
-        end
-        return s
+        return join(str_value(a) for a in x)
     end
 end
```

Comment:
@pfitzseb do you mind moving the definition of `str_value` to after `toplevel_binding` ? I want keep related code aggregated if possible.

Reply:
NVM, should be done in another PR.